model/project.go

```diff
@@ -1165,6 +1170,26 @@ func (p *Project) FindTaskGroup(name string) *TaskGroup {
 	return nil
 }
 
+func FindContainerConfigurationForTask(t task.Task) (*Container, error) {
```

Comment:
* `FindContainerFromProject`?
* Documentation for exported methods.
* Is this used anywhere? Is it supposed to be used when resolving whether it's running on a container vs. distro?

Reply:
It is not currently used since there is no use right now for actually applying the container configs defined in YAML. I just added this since finding a container configuration from a task's Container string is something that will be required in the future.

Reply:
I'm fine with leave it in then. Can we still write unit tests for it?